<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewBook</class>
 <widget class="QDialog" name="addNewBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 74, 106);</string>
  </property>
  <widget class="QLineEdit" name="newBookName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 181, 205);
color: rgb(255, 255, 255);
border: 0px groove gary;
border-radius:5px;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddNewBook">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>168</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 181, 205);
color: rgb(255, 255, 255);
border: 0px groove gary;
border-radius:5px;
selection-background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(162, 181, 205);
color: rgb(255, 255, 255);
border: 0px groove gary;
border-radius:5px;
selection-background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>请输入作品名称：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
